tests/ui/dbg_macro/dbg_macro.rs

```diff
@@ -49,6 +41,7 @@ fn issue9914() {
     macro_rules! expand_to_dbg {
         () => {
             dbg!();
+            //~^ ERROR: the `dbg!` macro is intended as a debugging tool
```

Comment:
had to put that ERROR comment inside this macro because I don't know where else to put it 😆